field_rrule/demo/res_partner.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Wow, you're breaking new ground there!
If I read it correctly, you are capable of conditionally import the code in `demo/`, right?
I think we should improve the conditions: the demo code should be loaded as long as the `demo` server option is enabled.